docs/index.md

```diff
@@ -1 +1,26 @@
+# Help index
 
+Please start by reading the [Manual](Manual.md)
+
+You may find yourself coming back to [the description of the Carp language](LanguageGuide.md)
+and to [its libraries.](Libraries.md)
+
+Also there is documentation about:
+[C Interop](CInterop.md); 
```

Comment:
At the moment this list of topics is rendered in html as one line, could you make it a list instead?

Reply:
good point, thank you. The source (md) is now updated.

Sadly, the python lib does not render it correctly, however cmark-cfm (by github) does.
One more reason to replace it very soon (I'm working on it),

Reply:
index.md is now rendered correctly in html:
that was a limitation in the python markdown library. took only 2 line breaks to work around it. :-/